admin/release/ubuntu/Dockerfile

```
@@ -77,7 +80,39 @@ RUN mkdir /usr/X11R6
 
 ## required for qt to find xcb-xlib deps
 RUN ln -s /usr/lib/x86_64-linux-gnu /usr/X11R6/lib64
+
+#RUN wget https://repo.anaconda.com/archive/Anaconda3-2024.10-1-Linux-x86_64.sh
+#RUN bash Anaconda3-2024.10-1-Linux-x86_64.sh -b -p /opt/anaconda3
+#RUN ls -l /opt/anaconda3
+#RUN /opt/anaconda3/bin/conda install -y -v -c conda-forge gcc=12.4.0
+#RUN /opt/anaconda3/bin/gcc --version
+#ENV PATH="/opt/anaconda3/bin:${PATH}"
+RUN gcc --version
+#RUN apt-get install -y clang
+RUN apt-get install -y libclang-dev
+RUN apt-get install -y gperf bison flex
+RUN apt-get install -y llvm
+RUN which llvm-config 
+
+## try to install python2 & put in path
+RUN wget https://repo.anaconda.com/archive/Anaconda2-2019.10-Linux-x86_64.sh
+RUN bash Anaconda2-2019.10-Linux-x86_64.sh -b -p /opt/anaconda2
+#RUN ls -l /opt/anaconda2/bin
+ENV PATH="/opt/anaconda2/bin:${PATH}"
+
+RUN apt-get install -y nodejs
+RUN apt-get install -y libnss3-dev
+RUN apt-get install -y libxkbfile-dev
+
 RUN cd qt-everywhere-src-$qt_version && env MAKEFLAGS=-j$parallel_compile ./configure -prefix /qt-$qt_version -release -opensource -confirm-license -platform linux-g++-64 -nomake tests -nomake examples -xcb -xcb-xlib -D QT_SHAREDMEMORY -D QT_SYSTEMSEMAPHORE -no-icu
+COPY mathops.h /qt-everywhere-src-$qt_version/qtwebengine/src/3rdparty/chromium/third_party/ffmpeg/libavcodec/x86/mathops.h
+RUN sed -i '6i #include \<cstdint\>' /qt-everywhere-src-$qt_version/qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/geometry.hpp
+RUN sed -i '7i #include \<cstdint\>' /qt-everywhere-src-$qt_version/qtlocation/src/3rdparty/mapbox-gl-native/include/mbgl/util/string.hpp
+RUN sed -i '4i #include \<cstdint\>' /qt-everywhere-src-$qt_version/qtlocation/src/3rdparty/mapbox-gl-native/src/mbgl/gl/stencil_mode.hpp
+
+RUN cd qt-everywhere-src-$qt_version && env MAKEFLAGS=-j$parallel_compile make -k ; echo 0
+RUN cd qt-everywhere-src-$qt_version && env MAKEFLAGS=-j$parallel_compile make
+RUN cd qt-everywhere-src-$qt_version && env MAKEFLAGS=-j$parallel_compile make -j$parellel_compile install
 RUN cd qt-everywhere-src-$qt_version && nohup make -j$parallel_compile 2>&1 > make.out
 RUN cd qt-everywhere-src-$qt_version && nohup make install -j1 2>&1 > makeinstall.out
 # RUN ls -l /qt-$qt_version/bin /qt-$qt_version/lib
```

```suggestion
RUN rm -r qt-everywhere-src-$qt_version
```
We don't need the source code of qt anymore after compiling it. That makes the resulting image 15 GB smalller.